database/events.api.php

```diff
@@ -106,4 +106,92 @@ function delete_event($event_id)
 function publish_event($event_id, $state)
 {
     return query_db("UPDATE deplacements SET open=? WHERE did=? LIMIT 1", $state, $event_id);
+}
+
+function save_registration($event_id, $user_id, $post, $competitions_id)
+{
+    if (isset($post['submit'])) {
+        $date = date('Y-m-d H:i:s', time());
+        #First check if the user wants to register to the event
+        if (isset($post['event_entry'])) {
+            query_db("REPLACE INTO inscriptions_depl(id_depl, id_runner, present, transport, heberg, courses, date, comment)
```

Comment:
Might be good to consider using the form validation utility tool I built. You can add new field types to it! If you feel unsure on how to do it I can refactor later, it's not a massive deal